cmd/commands/ceph.go

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
What about `cmd/ceph.go`? The `commands` subfolder seems redundant.

Reply:
the commands where not working if I had main.go and all other command files. so keeping as it for now.